<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 5 Multivariate statistics | The STRAF Book</title>
<meta name="author" content="Alexandre Gouy and Martin Zieger">
<meta name="description" content="In this chapter, we will describe the two multivariate statistics methods implemented in STRAF, Principal Component Analysis (PCA) and Multidimensional Scaling (MDS).  5.1 Principal Component...">
<meta name="generator" content="bookdown 0.25 with bs4_book()">
<meta property="og:title" content="Chapter 5 Multivariate statistics | The STRAF Book">
<meta property="og:type" content="book">
<meta property="og:description" content="In this chapter, we will describe the two multivariate statistics methods implemented in STRAF, Principal Component Analysis (PCA) and Multidimensional Scaling (MDS).  5.1 Principal Component...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 5 Multivariate statistics | The STRAF Book">
<meta name="twitter:description" content="In this chapter, we will describe the two multivariate statistics methods implemented in STRAF, Principal Component Analysis (PCA) and Multidimensional Scaling (MDS).  5.1 Principal Component...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">The STRAF Book</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Preface</a></li>
<li><a class="" href="introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="importing-data.html"><span class="header-section-number">2</span> Importing data</a></li>
<li><a class="" href="forensic-parameters.html"><span class="header-section-number">3</span> Forensic parameters</a></li>
<li><a class="" href="popgen-indices.html"><span class="header-section-number">4</span> Population genetics indices</a></li>
<li><a class="active" href="pca-mds.html"><span class="header-section-number">5</span> Multivariate statistics</a></li>
<li><a class="" href="ref-pop.html"><span class="header-section-number">6</span> Reference populations analysis</a></li>
<li><a class="" href="file-conversion.html"><span class="header-section-number">7</span> File conversion</a></li>
<li><a class="" href="snp-analysis.html"><span class="header-section-number">8</span> Analysing SNP data</a></li>
<li><a class="" href="local-computer.html"><span class="header-section-number">9</span> Running STRAF on a local computer</a></li>
<li><a class="" href="conclusion.html"><span class="header-section-number">10</span> Conclusion</a></li>
<li><a class="" href="references.html"><span class="header-section-number">11</span> References</a></li>
<li><a class="" href="acknowledgments.html"><span class="header-section-number">12</span> Acknowledgments</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="technical-details.html"><span class="header-section-number">A</span> Technical details</a></li>
<li><a class="" href="straf-commitments.html"><span class="header-section-number">B</span> STRAF’s commitments</a></li>
</ul>

        <div class="book-extra">
          
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="pca-mds" class="section level1" number="5">
<h1>
<span class="header-section-number">5</span> Multivariate statistics<a class="anchor" aria-label="anchor" href="#pca-mds"><i class="fas fa-link"></i></a>
</h1>
<p>In this chapter, we will describe the two multivariate statistics methods implemented
in STRAF, <strong>Principal Component Analysis</strong> (<strong>PCA</strong>) and <strong>Multidimensional Scaling</strong> (<strong>MDS</strong>).</p>
<div id="principal-component-analysis-pca" class="section level2" number="5.1">
<h2>
<span class="header-section-number">5.1</span> Principal Component Analysis (PCA)<a class="anchor" aria-label="anchor" href="#principal-component-analysis-pca"><i class="fas fa-link"></i></a>
</h2>
<p>PCA is a method of <strong>dimensionality reduction</strong>. The principle is that it tries to
<strong>capture most of the variation</strong> present in the data and <strong>project</strong> it onto a
small number of new axes called <strong>principal components</strong> (PCs).</p>
<p>This is a useful method to capture variation from a large number of variables and
allows to discover hidden patterns by increasing interpretability.</p>
<p>In our case, if we consider that <strong>each allele at each locus</strong> is a variable, and that
our individual observations are the presence / absence of each allele for each sample,
we end up with a highly dimensional dataset (we have as many variables as we have
alleles!). It gets even worse if you analyse genome sequences, where you can have millions
of variables in your dataset. This is definitely not an interpretable dataset if you
are not able to easily extract relevant information.</p>
<p>PCA allows to bring most of the variation existing between our samples onto a few
axes capturing most of the variation (PCs).</p>
<p>In STRAF, you can perform a PCA by going into the <strong>PCA - MDS</strong> tab and checking
the following <em>Run and plot a PCA</em> checkbox.</p>
<div class="capture">
<div class="inline-figure"><img src="img/capture_pca_1.png" width="100%"></div>
</div>
<p>It will trigger the PCA computation and return a graph. You are also able to
download the coordinates of the samples on different PCs (also called eigenvectors).</p>
<div class="interpretation">
<p><strong>Interpreting PCA results</strong></p>
<p>PCA has become a popular tool in population genetics, as the relationships between
individuals on the PCA projection tend to reflect their genetic <strong>relatedness</strong>. The
closer individuals are on the PCA projection, the more genetically related they tend to be.</p>
<p>It can also be used as a <strong>quality control</strong> tool. For example, if a sample is very far
from all the others, including the ones that are part of the same population,
it is possible that there is an issue with the data. One would need to inspect
the raw data and check if any strange pattern can be found.</p>
<p>Also, PCA with <strong>haploid markers</strong> can typically reveal clustering according to haplogroups.</p>
<p>It is important to be aware of the influence of non-demographic processes on
the PCA projection. For example, <strong>imbalanced sample sizes</strong> between populations
can drive some patterns. When populations are sampled unevenly, the projection will
be <strong>distorted</strong> and distances observed on the projection can be driven by these
differences and not by their evolutionary history.</p>
<p>Hence, as multiple processes influence the results, PCA should remain an
<strong>exploratory approach</strong> and further analyses should be conducted before drawing
any major conclusions on the relationships between populations and individuals.</p>
</div>
</div>
<div id="multidimensional-scaling-mds" class="section level2" number="5.2">
<h2>
<span class="header-section-number">5.2</span> Multidimensional Scaling (MDS)<a class="anchor" aria-label="anchor" href="#multidimensional-scaling-mds"><i class="fas fa-link"></i></a>
</h2>
<p>An <strong>MDS</strong> is conceptually similar to a PCA. One of the main differences is
that it requires input data to be in a different format. As PCA uses raw genotypes and
can accommodate data at the individual level, <strong>pairwise distances</strong> between data
points are used for an MDS.</p>
<p>In forensics, it is typically used to compare <strong>populations</strong> and not
<strong>individuals</strong>, even though it would be theoretically possible.</p>
<p>As <strong>pairwise distances between populations</strong> are used, and MDS can be based on
any genetic distance: one could for example compute pairwise <strong>FSTs</strong>,
<strong>Nei’s genetic distance</strong>, or any other metric.</p>
<p>Based on these distances, the MDS will <strong>project</strong> the populations onto a
<strong>lower-dimensional space</strong>. This projection facilitates the interpretation of
relationships between populations.</p>
<div class="capture">
<div class="inline-figure"><img src="img/capture_mds_1.png" width="100%"></div>
</div>
<div class="interpretation">
<p><strong>Interpreting MDS results</strong></p>
<p>Just like with a PCA, the <strong>closer</strong> populations are on the MDS projection,
the more <strong>genetically related</strong> they tend to be, based on the markers that have been
used to compute their pairwise genetic distances.</p>
<p>In forensics, the MDS is often used as a confirmation that the population
sample clusters with the expected populations, e.g. a sample from Switzerland
with other western European populations.</p>
<p>Again, like for the PCA, the MDS should remain an
<strong>exploratory approach</strong> and further analyses should be conducted before drawing
any major conclusions on the relationships between populations and individuals.</p>
</div>

</div>
</div>
  <div class="chapter-nav">
<div class="prev"><a href="popgen-indices.html"><span class="header-section-number">4</span> Population genetics indices</a></div>
<div class="next"><a href="ref-pop.html"><span class="header-section-number">6</span> Reference populations analysis</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#pca-mds"><span class="header-section-number">5</span> Multivariate statistics</a></li>
<li><a class="nav-link" href="#principal-component-analysis-pca"><span class="header-section-number">5.1</span> Principal Component Analysis (PCA)</a></li>
<li><a class="nav-link" href="#multidimensional-scaling-mds"><span class="header-section-number">5.2</span> Multidimensional Scaling (MDS)</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
          
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>The STRAF Book</strong>" was written by Alexandre Gouy and Martin Zieger. It was last built on 2022-07-08.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
