<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 8 Analysing SNP data | The STRAF Book</title>
<meta name="author" content="Alexandre Gouy and Martin Zieger">
<meta name="description" content="8.1 STRs vs. SNPs STRs have been and remain the dominant marker used in forensic genetics. This is mainly due to the fact that STRs have a high mutation rate, therefore are more diverse in human...">
<meta name="generator" content="bookdown 0.25 with bs4_book()">
<meta property="og:title" content="Chapter 8 Analysing SNP data | The STRAF Book">
<meta property="og:type" content="book">
<meta property="og:description" content="8.1 STRs vs. SNPs STRs have been and remain the dominant marker used in forensic genetics. This is mainly due to the fact that STRs have a high mutation rate, therefore are more diverse in human...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 8 Analysing SNP data | The STRAF Book">
<meta name="twitter:description" content="8.1 STRs vs. SNPs STRs have been and remain the dominant marker used in forensic genetics. This is mainly due to the fact that STRs have a high mutation rate, therefore are more diverse in human...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">The STRAF Book</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Preface</a></li>
<li><a class="" href="introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="importing-data.html"><span class="header-section-number">2</span> Importing data</a></li>
<li><a class="" href="forensic-parameters.html"><span class="header-section-number">3</span> Forensic parameters</a></li>
<li><a class="" href="popgen-indices.html"><span class="header-section-number">4</span> Population genetics indices</a></li>
<li><a class="" href="pca-mds.html"><span class="header-section-number">5</span> Multivariate statistics</a></li>
<li><a class="" href="ref-pop.html"><span class="header-section-number">6</span> Reference populations analysis</a></li>
<li><a class="" href="file-conversion.html"><span class="header-section-number">7</span> File conversion</a></li>
<li><a class="active" href="snp-analysis.html"><span class="header-section-number">8</span> Analysing SNP data</a></li>
<li><a class="" href="local-computer.html"><span class="header-section-number">9</span> Running STRAF on a local computer</a></li>
<li><a class="" href="conclusion.html"><span class="header-section-number">10</span> Conclusion</a></li>
<li><a class="" href="references.html"><span class="header-section-number">11</span> References</a></li>
<li><a class="" href="acknowledgments.html"><span class="header-section-number">12</span> Acknowledgments</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="technical-details.html"><span class="header-section-number">A</span> Technical details</a></li>
<li><a class="" href="straf-commitments.html"><span class="header-section-number">B</span> STRAF’s commitments</a></li>
</ul>

        <div class="book-extra">
          
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="snp-analysis" class="section level1" number="8">
<h1>
<span class="header-section-number">8</span> Analysing SNP data<a class="anchor" aria-label="anchor" href="#snp-analysis"><i class="fas fa-link"></i></a>
</h1>
<div id="strs-vs.-snps" class="section level2" number="8.1">
<h2>
<span class="header-section-number">8.1</span> STRs vs. SNPs<a class="anchor" aria-label="anchor" href="#strs-vs.-snps"><i class="fas fa-link"></i></a>
</h2>
<p>STRs have been and remain the dominant marker used in forensic genetics.</p>
<p>This is mainly due to the fact that STRs have a high mutation rate, therefore
are more diverse in human populations. This explains their high <strong>power of discrimination</strong>.
Furthermore, they can also be used in deciphering mixture components, a very common
case in forensics. Finally, they can be combined in multiplex assays, which is convenient when lowamounts of biological material can be recovered.</p>
<p>Even though STRs will still remain the standard markers in forensic genetics
for many years, the application of sequencing data, especially the analyses
of SNPs became more and more popular in recent years.</p>
<p>Indeed, it nowadays easier and cheaper to generate whole-genome sequences than before.
One could wonder why STRs are still so popular in forensic practice, and have not been replaced by SNPs that are easily generated from Next-Generation Sequencing (NGS) data.</p>
<p>The advantages of SNPs are a higher multiplex capacity (with STRs and CE we can
analyse not much more than 25 markers in parallel) and therefore a higher
discriminative power and shorter amplicon size (interesting for degraded samples).</p>
</div>
<div id="how-to-analyse-snp-data-in-straf" class="section level2" number="8.2">
<h2>
<span class="header-section-number">8.2</span> How to analyse SNP data in STRAF?<a class="anchor" aria-label="anchor" href="#how-to-analyse-snp-data-in-straf"><i class="fas fa-link"></i></a>
</h2>
<p>It is in theory possible to use any marker type in STRAF, as long as the input
format corresponds to what is expected from the software, that is:</p>
<ul>
<li><p>2 columns per locus for diploid data, 1 for haploid;</p></li>
<li><p>alleles are encoded by a number: for example, a diallelic SNP could be encoded as
100 for the reference allele, and 101 for the alternative allele.</p></li>
</ul>
<p>The main challenge remains encoding these values and converting from the initial
data format. As an example, a genotype table is easier to convert than raw sequences
or bioinformatics-specific formats such as the Variant Calling Format (VCF).</p>

</div>
</div>
  <div class="chapter-nav">
<div class="prev"><a href="file-conversion.html"><span class="header-section-number">7</span> File conversion</a></div>
<div class="next"><a href="local-computer.html"><span class="header-section-number">9</span> Running STRAF on a local computer</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#snp-analysis"><span class="header-section-number">8</span> Analysing SNP data</a></li>
<li><a class="nav-link" href="#strs-vs.-snps"><span class="header-section-number">8.1</span> STRs vs. SNPs</a></li>
<li><a class="nav-link" href="#how-to-analyse-snp-data-in-straf"><span class="header-section-number">8.2</span> How to analyse SNP data in STRAF?</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
          
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>The STRAF Book</strong>" was written by Alexandre Gouy and Martin Zieger. It was last built on 2022-07-08.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
